src/main/java/frc/robot/commands/operatorcommands/TeleClimbTilt.java

```diff
@@ -0,0 +1,30 @@
+package frc.robot.commands.operatorcommands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.climber.Climber;
+
+public class TeleClimbTilt extends CommandBase {
+
+    // r 16
+
+    private final Climber climber;
+
+    public TeleClimbTilt(Climber climber) {
+        this.climber = climber;
+        climber.setBrakeMode(true);
```

Comment:
You should not be setting brakemode in the constructor. This should either be done in `RobotContainer`, or in the command's initialize method. But probably in robot container.